docs/extending/custom_icons.md

```diff
@@ -54,3 +57,35 @@ return (
    </svg>
 )
 ```
+
+### Icons React component
+
+You can use React component to change icons in Back Office and Page Builder.
+
+See the example with React component in the `alert.js` configuration:
```

Comment:
It would be nice to have an explanation of the example, sthg like: "The followin example from the `alert.js` configuration shows XYZ".